source/_tags/tagsindex.rst

```diff
@@ -0,0 +1,11 @@
+:orphan:
```

Comment:
Question: Why are we collecting orphan pages? My assumption is because this is a strict migration, and we can clean up things tagged "orphan" at a later point